website/static/service-worker.js

```diff
@@ -15,33 +17,33 @@ toolbox.precache([
   "lib/parsers-location.js",
   ...plugins,
   "playground.js",
-  "https://unpkg.com/sw-toolbox@3.6.0/sw-toolbox.js",
+  "https://cdnjs.cloudflare.com/ajax/libs/sw-toolbox/3.6.1/sw-toolbox.js",
```

Comment:
You probably know that but still worth flagging that `sw-toolbox` is deprecated:
https://github.com/GoogleChromeLabs/sw-toolbox

Also 3.6.1 does not exist on NPM (https://www.npmjs.com/package/sw-toolbox/v/3.6.1), only on GitHub (https://github.com/GoogleChromeLabs/sw-toolbox/releases/tag/v3.6.1). Not sure where CloudFlare pulls the artifact from, but it seem to be doing that correctly 🤔

Reply:
Downgraded to 3.6.0

Reply:
https://github.com/prettier/prettier/issues/8763#issuecomment-663385515